packages/textInput/components/TextInput.tsx

```diff
@@ -65,13 +76,15 @@ export class TextInput<
 
   public render() {
     const labelContent = this.getLabelContent();
+    const tooltipContent = this.getTooltipContent();
     const containerProps: { className?: string } = {};
     if (this.props.className) {
       containerProps.className = this.props.className;
     }
     return (
       <div {...containerProps}>
         {labelContent}
+        {tooltipContent}
```

Comment:
We get nicer vertical alignment if we use a flexbox container to wrap both the `<label>` and the `<Tooltip>`, and move the bottom margin from the `<label>` to the flexbox container:
```
<div className={cx(margin("bottom", "xxs"), flex({ align: "center" }))}>
    {labelContent}
    {tooltipContent}
</div>
```

Then, we can also get rid of the ternary in `getLabelContent()`: https://github.com/dcos-labs/ui-kit/pull/327/files#diff-d6ddbf113c06302e95d5b248c029f35fR106